.github/workflows/create-release-build.yaml

```diff
@@ -0,0 +1,72 @@
+# This starter workflow is for a CMake project running on a single platform.
+name: Create Release Build
+
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+  # Manual trigger using the Actions page.
+  workflow_dispatch:
+
+env:
+  # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
+  BUILD_TYPE: Release
+
+jobs:
+  build_release:
+    runs-on: ubuntu-22.04-8core
+
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        submodules: 'recursive'
+        ref: '${{github.ref}}'
+
+    - name: Install Dependencies
+      run: sudo apt-get -y install libunwind-dev libgflags-dev libssl-dev libelf-dev protobuf-compiler libzstd-dev
+
+    - name: Configure CMake
+      # Configure CMake in a 'build' subdirectory. `CMAKE_BUILD_TYPE` is only required if you are using a single-configuration generator such as make.
+      # See https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html?highlight=cmake_build_type
+      run: cmake -B ${{github.workspace}}/build -DCMAKE_BUILD_TYPE=${{env.BUILD_TYPE}} -DENABLE_TOOL=LLVM -S ${{github.workspace}}
+
+    - name: Build
+      # Build your program with the given configuration
+      run: make -C ${{github.workspace}}/build -j6
+
+    - name: Run Tests
+      run: make -C ${{github.workspace}}/build test
+
+    # - uses: actions/upload-artifact@v4
+    #   with:
+    #     name: create_llvm_prof.zip
+    #     path: ${{github.workspace}}/build/create_llvm_prof
+
+    - name: Create Release
+      id: create_release
+      uses: actions/create-release@v1
+      env:
+        GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+      with:
+        tag_name: ${{github.ref}}
+        release_name: ${{github.ref_name}}
+        body_path: "${{github.workspace}}/RELEASE.md"
+        draft: false
+        prerelease: false
+
+    - name: Create Zip Package
+      run: |
+        zip --junk-paths create_llvm_prof-x86_64-${{github.ref_name}}.zip ${{github.workspace}}/build/create_llvm_prof
```

Comment:
I guess it's safe to assume that the runner will always be x86?

Reply:
Yes, that's the case. "ubuntu-arm-22.04-8core" is x86_64. We also have "arm-ubuntu-arm-22.04-8core", which is for arm.